src/core/dep/jquery-fix.js

```diff
@@ -163,6 +163,33 @@ jQuery.htmlPrefilter = function( html ) {
  * This implementation leverage DOMPurify for filtering every string prior DOM manipulation by jQuery
  *
  */
+
+// START: add hooks to DOMPurify to allow external links when they meet certain conditions as defined here: https://owasp.org/www-community/attacks/Reverse_Tabnabbing
+DOMPurify.addHook( "beforeSanitizeAttributes", function( node ) {
+
+	// Add "data-external-link" to all <a> with a target="_blank" and rel="noreferrer"
+	if (
+		node.tagName === "A" &&
+		node.getAttribute( "target" ) &&
+		node.getAttribute( "target" ) === "_blank" &&
+		node.getAttribute( "rel" ) &&
+		node.getAttribute( "rel" ).includes( "noreferrer" )
```

Comment:
Using `.includes()` function is problematic because the following value in the `rel` attribute return true:
* `noreferrers`
* `anoreferrer`
* `snoreferrers` (Add this example as a unit test)

I suggest to do instead and you would not require to check if that attribute is set or not.
```suggestion
		node.relList.contains( "noreferrer" )
```

Reply:
Merci pour la suggestion!